src/Tribe/Admin/Notices.php

```diff
@@ -329,4 +331,60 @@ function maybe_display_paystack_notice() {
 		);
 
 	}
+
+	/**
+	 * Display AR Page notice for FSE theme detected.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	public function maybe_display_fse_ar_page_notice() {
+		// Bail if we aren't in Tickets > Settings.
+		if ( Settings::$settings_page_id !== tec_get_request_var( 'page' ) ) {
+			return;
+		}
+
+		// Bail if Tickets Plus is not active.
+		if ( ! did_action( 'tec_container_registered_provider_Tribe__Tickets_Plus__Service_Provider' ) ) {
```

Comment:
I really like Stephen's approach regarding that and would love to be consistent accross these checks. [Here](https://lw.slack.com/archives/C01H7Q57P1C/p1725247304646939?thread_ts=1725247070.454189&cid=C01H7Q57P1C) is the pinned message in slack